packages/objects/src/interfaces/ISDQLQueryObject.ts

```diff
@@ -15,6 +18,8 @@ export interface ISDQLQueryObject {
   timestamp: ISO8601DateString;
   expiry: ISO8601DateString;
   description: string;
+  image?: IpfsCID;
+  points?: number;
```

Comment:
Is the concept of points now separate from compensations, and is there a guarantee that, since we can mint a very large number of ERC20 tokens and have the ability to mint more, preview calls are not required in this case? Also, do individual questionnaires not have points, whereas only offers do have points?

Reply:
The points here is a display only property. This is intended to power the the "100 Points" tag at the bottom of an offer. Muhammed, I recommend properties `rewardTag: string`, `rewardContract: EVMContractAddress`, and `rewardChain: ChainId`

Reply:
In the context of offers, the tag at the bottom is supposed to be actual on-chain points, not fake ones- but the problem is that rewards aren't that bloody simple. So just some generic information like I suggest above could populate it as a sort of hint.